packs/trident-operator-v23.01.0/pack.json

```diff
@@ -0,0 +1,17 @@
+{
+  "addonType":"system app",
+  "cloudTypes": [
+    "all"
+  ],
+  "annotations": {
+    "docsURL": "https://github.com/NetApp/trident",
+    "description": "NetApp Trident"
```

Comment:
Could you add additional annotations below to this pack? This will help Palette UI to differentiate between in-house vs community maintained packs.
```
     "source": "community",
     "contributor" : "spectrocloud"
```